blobstore/src/test/java/org/jclouds/blobstore/integration/internal/BaseContainerIntegrationTest.java

```diff
@@ -195,39 +197,55 @@ public void testClearWithOptions() throws InterruptedException {
          options.prefix("path/1/2/3");
          options.recursive();
          view.getBlobStore().clearContainer(containerName, options);
-         assertConsistencyAwareContainerSize(containerName, 2);
+         view.getBlobStore().countBlobs(containerName);
```

Comment:
Why call `countBlobs` if you don't check the value?  Repeated below.

Reply:
removed